src/screens/FormScreen.js

```diff
@@ -5,6 +5,18 @@ const FormScreen = ({ navigation }) => {
   const [restaurantType, setRestaurantType] = useState('')
   const [travelType, setTravelType] = useState('')
   const [cost, setCost] = useState('')
+  const [userLocation, setUserLocation] = useState({})
+
+  const findUserCoordinates = () => {
+    navigator.geolocation.getCurrentPosition(
+      position => {
```

Comment:
Is this how a user's current location will get translated into latitude/longitude and sent back? I see it's on the form page but wasn't sure. Thanks!